docusaurus/docs/manage/02-router/01-overview.md

```diff
@@ -0,0 +1,63 @@
+---
+id: router-deploying-overview
+title: Deployment Overview
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import styles from './styles.module.css';
+
+In this article, we are describing the process of router deployment and what attributes can be updated, removed, added after the deployment is completed.  
+
+## Components of Deployment
+
+:::info Notes
+Currently, the DNS name or IP address can not be changed after the deployment, because it requires to update the `Sans` section of the router server certificate. To do that one would need to re-enroll the router, which can not be done without creating a new router.

```

Comment:
Personally, I would favor leaving out extra details that aren't explicitly necessary like the SANS stuff. Ideally, we'd point people to a link where that whole thing is described but we don't have that yet, so I'd just leave that detail out

**From:**
Currently, the DNS name or IP address can not be changed after the deployment.

**To:**
Changing a router's advertised DNS entry or IP address (if not DNS-based) is not supported. To change these values after enrollment, the router must be deleted and re-enrolled

It's basically the same but doesn't get into the SANS bits which I expect would confuse all but the hardest core of readers

Reply:
makes sense, will update it